docs/content/docs/deployment/adaptive_batch_scheduler.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Adaptive Batch Scheduler
+weight: 5
+type: docs
+
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Adaptive Batch Scheduler
+
+The Adaptive Batch Scheduler can automatically decide parallelisms of job vertices for batch jobs. If a job vertex is not set with a parallelism, the scheduler will decide parallelism for the job vertex according to the size of its consumed datasets. This can bring many benefits:
+- Batch job users can be relieved from parallelism tuning
+- Automatically tuned parallelisms can be vertex level and can better fit consumed datasets which have a varying volume size every day
+- Vertices from SQL batch jobs can be assigned with different parallelisms which are automatically tuned
```

Comment:
What's the target audience? Does regular Flink user supposed to know what the job vertex is? Overall this page feels bit too low level 🤔.

On the other hand I don't think that other pages withing this section are all much better in this regard 🤔

Reply:
> What's the target audience? Does regular Flink user supposed to know what the job vertex is? Overall this page feels bit too low level 🤔.

Thanks for pointing that out. Maybe `stage` is more appropriate？


> On the other hand I don't think that other pages withing this section are all much better in this regard 🤔

I'll check the rest content.

Reply:
Or use `operator`, although it is not exactly the same as the job vertex.

Reply:
+ 1 for `operator`. It is the concept that users can/must understand. I think adaptively deciding parallelisms does mean to adaptively deciding parallelisms for operators. We just do not want to break beneficial operator chaining, so that parallelisms are decided for OperatorChain/JobVertex.